docs/VnVPlan/VnVPlan.tex

```diff
@@ -97,12 +101,20 @@ \subsection{Objectives}
 
 \subsection{Relevant Documentation}
 
-\wss{Reference relevant documentation.  This will definitely include your SRS
-	and your other project documents (design documents, like MG, MIS, etc).  You
-	can include these even before they are written, since by the time the project
-	is done, they will be written.}
+Below are the relevant documents that will be referred to in the VnV Plan.
```

Comment:
I assume `VnV` will have be added to `Symbols, Abbreviations and Acronyms` if we are use that (I don't think it will be used anywhere else so we might just want to spell it out).